types/components/Spinner.d.ts

```diff
@@ -19,7 +19,7 @@ export interface SpinnerProps {
 }
 
 declare class Spinner<
-  As extends React.ElementType = typeof Spinner
+  As extends React.ElementType = 'div'
```

Comment:
oops

Reply:
You mean is my error? I don’t think makes any sense that Spinner passes props to another Spinner, so my fix ok. Am I right?

Reply:
no, i mean on our end. this was a silly typo